packages/backend/src/data/schema/api-def.schema.ts

```diff
@@ -39,6 +39,10 @@ const apiDefSchema = new mongoose.Schema(
     request_complexity_limit: Number,
     rate_limit: Object,
     mesh: Object,
+    webhooks: {
+      type: Array,
```

Comment:
an array of what? should be specified

Reply:
update tests within test/common: apiDefExpample and schema

Reply:
actually, it should be `mongoCipherType`, as webhook.url could contain some private keys